radicle-cli/src/commands/comment.rs

```diff
@@ -22,11 +23,12 @@ pub const HELP: Help = Help {
     usage: r#"
 Usage
 
-    rad comment <id> [-m <text>]
+    rad comment <id> [options...]
 
 Options
 
     -m, --message               Comment message
+        --reply-to <id>         Index of comment writing a reply for
```

Comment:
Something like this maybe:
```suggestion
        --reply-to <comment>        Reply to a comment
```